spec/jobs/access_request_satisfaction_survey_job_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe AccessRequestSatisfactionSurveyJob do
+  before do
+    expect(JwtApiEntreprise::Operation::AccessRequestSatisfactionSurvey).to receive(:call)
```

Comment:
Je trouve ça bizarre d'avoir un `expect` dans un `before` (où l'on ne définit pas à proprement parler d'expectation).

Ce que je propose:

```suggestion
    allow(JwtApiEntreprise::Operation::AccessRequestSatisfactionSurvey).to receive(:call)
```

Et plus bas:

```ruby
  it 'calls the operation' do
    expect(JwtApiEntreprise::Operation::AccessRequestSatisfactionSurvey).to receive(:call)

    subject.perform_now
  end
```

Reply:
Je déplace l'`expect` vers le bloc `it` :

```ruby
subject { described_class }

it 'calls the operation' do
  expect(JwtApiEntreprise::Operation::AccessRequestSatisfactionSurvey).to receive(:call)
  subject.perform_now
end
```

Si ça te va, je propose de ne pas ajouter `allow`, l'objet n'étant pas mocké.

Reply:
Nous pourrions aussi l'écrire comme ça si vous préférez :

```ruby
subject { described_class }

it 'calls the operation' do
  expect(JwtApiEntreprise::Operation::AccessRequestSatisfactionSurvey).to receive(:call)
end

after do
  subject.perform_now
end
```

Une seule ligne par bloc `it`, je trouve ça bien perso.

Reply:
J'ai une préférence pour la 1e version, mais as u ouiche.